src/components/common/Table/Table.js

```diff
@@ -0,0 +1,313 @@
+import React, { useState } from 'react';
+import { TagsComponent } from '../index';
+import {
+  Table,
+  Input,
+  Dropdown,
+  Menu,
+  Typography,
+  Form,
+  Tag,
+  Space,
+  Popconfirm,
+} from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+
+const users = [
+  {
+    name: 'Ruben Ramirez',
+    role: 'Administrator',
+    programs: ['Sheltering', 'Aftercare', 'Prevention'],
+  },
+  {
+    name: 'Michael Scott',
+    role: 'Program Manager',
+    programs: ['Prevention'],
+  },
+  {
+    name: 'John Legend',
+    role: 'Program Manager',
+    programs: ['Prevention', 'Aftercare', 'Sheltering'],
+  },
+  {
+    name: 'Mary Higgins',
+    role: 'Service Provider',
+    programs: ['Sheltering', 'Aftercare'],
+  },
+  {
+    name: 'Bilbo Baggins',
+    role: 'Program Manager',
+    programs: ['Prevention', 'Aftercare', 'Sheltering'],
+  },
+  {
+    name: 'Frodo Baggins',
+    role: 'Service Provider',
+    programs: ['Sheltering', 'Aftercare'],
+  },
+  {
+    name: 'Ruben Ramirez',
+    role: 'Administrator',
+    programs: ['Sheltering', 'Aftercare', 'Prevention'],
+  },
+  {
+    name: 'Michael Scott',
+    role: 'Program Manager',
+    programs: ['Prevention', 'Sheltering', 'Aftercare'],
+  },
+  {
+    name: 'John Legend',
+    role: 'Program Manager',
+    programs: ['Prevention', 'Aftercare', 'Sheltering'],
+  },
+  {
+    name: 'Mary Higgins',
+    role: 'Service Provider',
+    programs: ['Sheltering', 'Aftercare'],
+  },
+  {
+    name: 'Bilbo Baggins',
+    role: 'Program Manager',
+    programs: ['Prevention', 'Aftercare', 'Sheltering'],
+  },
+  {
+    name: 'Frodo Baggins',
+    role: 'Service Provider',
+    programs: ['Sheltering', 'Aftercare'],
+  },
+];
```

Comment:
The user data will be fetched from backend in future but for first run this is good